apps/site/assets/ts/stop/components/StationInformation.tsx

```diff
@@ -10,6 +10,10 @@ import AccessibilityAmenityCard from "./amenities/AccessibilityAmenityCard";
 import FareSalesAmenityCard from "./amenities/FareSalesAmenityCard";
 import { alertsByActivity } from "../../models/alert";
 
+const getParkingAlerts = (alerts: Alert[]): Alert[] => {
```

Comment:
suggestion: this should be in `alert.ts` with the other alert filtering functions